### DISCUSSION TITLE
what does Riienrollment download zip do on  devtools://devtools/bundled/devtools_app.html?experiments=true&wss=rig.kxtz.dev/  ,the update gui one ?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1378] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
?

**Comment by akabutnicer** [comment, marked as answer]
it files your device and user policies

**Reply by ghost** [reply]
Is it useful?

**Reply by Charlieis1nsan3** [reply]
yes. kinda like Pollen. pollen was a exploit that allowed users to edit chrome policy. so you can just unblock stuff like chrome://extensions (if blocked) and other stuff too.

**Reply by ghost** [reply]
how do i do it or know it works(ed)

**Reply by S-PScripts** [reply]
you gotta actually try it to know if it works

**Reply by akabutnicer** [reply]
@Charlieis1nsan3 yeah, just the opposite. instead of editing policies, you READ the policies. so to answer your question, if you know what you are doing then I guess it could be useful for other purposes. but generally, nah.

**Reply by Charlieis1nsan3** [reply]
oops. lol. dont judge. I never got to use Pollen or Rigtools
